go/common/sgx/pcs/certificates.go

```diff
@@ -0,0 +1,52 @@
+package pcs
+
+import (
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+)
+
+const pcsTrustRootCert = `-----BEGIN CERTIFICATE-----
```

Comment:
I had a [build script](https://github.com/oasislabs/dcap-quote-verification/blob/master/build.rs#L32) fetch and inline this automatically. The API version went up by two since then, though.

Reply:
Certs changing (especially extremely long lived ones like Intel promises) should be viewed with suspicion, not automated away.